lib/Doctrine/ruleset.xml

```diff
@@ -380,6 +380,7 @@
             </property>
         </properties>
     </rule>
+    <rule ref="SlevomatCodingStandard.TypeHints.PropertyTypeHintSpacing"/>
```

Comment:
The comment for this rule and the others are in lines 360-362:

https://github.com/doctrine/coding-standard/blob/d025e82a6ac1248d0333a92af1e50041a9385a2f/lib/Doctrine/ruleset.xml#L360-L362